packages/tools/src/tools/annotation/LivewireContourTool.ts

```diff
@@ -928,22 +924,29 @@ class LivewireContourTool extends ContourSegmentationBaseTool {
       return;
     }
 
-    const { annotation, sliceToWorld } = this.editData;
+    const { annotation, sliceToWorld, worldToSlice, closed, newAnnotation } =
+      this.editData;
     let { pointArray: imagePoints } = livewirePath;
 
     if (imagePoints.length > 1) {
       imagePoints = [...imagePoints, imagePoints[0]];
     }
 
+    // Save the annotation in clockwise winding direction only after closing it
```

Comment:
Thank you for the comment - I appreciate that because I wasn't sure how you were going to handle this one.